stubs/django/template/base.pyi

```diff
@@ -0,0 +1,4 @@
+from typing import Union
```

Comment:
We need a copy of the open source license at the top of these files

Reply:
added to base and engine @gbleaney